### PR TITLE
chore: push packages to install branch with travis

### LINKED ISSUE #467: Add stable installer for deb and snap on install branch
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current status?**
- The install branch does not have any files

* **What is the expected behaviour?**
- The installers should be added to the install branch like that in the android repo

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #467

### CODE REVIEW

**Comment by adityastic** [comment]
@CloudyPadmal @mariobehling add the GITHUB_API_KEY environment variable and merge this. You should then get your installable packages in install branch

**Comment by adityastic** [comment]
@AakashMallik review

File: .travis.yml [hunk 1]
```diff
@@ -1,58 +1,9 @@
...
-    - npm test -- --coverage
-
-# test stage - lint job
-lint:
-  stage: test
-  script:
-    - npm run lint
+  - npm run build
+after_success:
+  - bash travis-scripts/upload-packages.sh
```
Comment (AakashMallik) [inline .travis.yml]: new line

File: travis-scripts/upload-packages.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+rm -rf *
+
+find ../dist -type f \( -name '*.snap' -o -name '*.deb' \) -exec cp -v {} . \;
+
+git checkout --orphan temporary
+git add --all .
+git commit -am "[Auto] Update Install Packages ($(date +%Y-%m-%d.%H:%M:%S))"
+git branch -D install
+git branch -m install
+git push origin install --force --quiet > /dev/null
```
Comment (AakashMallik) [inline travis-scripts/upload-packages.sh]: new line

**Pull request merged**